packages/MainUI/components/Table/index.tsx

```diff
@@ -53,6 +55,7 @@ const DynamicTableContent = memo(function DynamicTableContent({ tab }: DynamicTa
     ({ row }: { row: MRT_Row<Record<string, unknown>> }) => ({
       onClick: () => {
         selectRecord(row.original as never, tab);
+        setSelectedRecord(row.original as never);
```

Comment:
I think we could rely in the metadata context and the selected records there, but I will check